src/controllers/dlgprefcontroller.cpp

```diff
@@ -406,6 +425,24 @@ QString DlgPrefController::mappingFileLinks(
 
         linkList << scriptFileLink;
     }
+
+#ifdef MIXXX_USE_QML
+    for (const auto& qmlLibrary : pMapping->getLibraryDirectories()) {
+        QString scriptFileLink = coloredLinkString(
+                m_pLinkColor,
+                qmlLibrary.dirinfo.fileName(),
+                qmlLibrary.dirinfo.absoluteFilePath());
+        if (!qmlLibrary.dirinfo.exists()) {
+            scriptFileLink +=
+                    QStringLiteral(" (") + tr("missing") + QStringLiteral(")");
```

Comment:
I think it's preferred to build a translatable string like `tr(" (missing)")` so the translator sees the full format context. (it's not about translating words per se). https://doc.qt.io/qt-5/i18n-source-translation.html.  Check to see how we do it elsewhere in mixxx

Reply:
I copied the syntax as it was used [here](https://github.com/mixxxdj/mixxx/blob/10cfd2fdc33d79dc8534a22748f3be7aaed841e4/src/controllers/dlgprefcontroller.cpp#L401) a few line above in the same file.

I will update accordingly, should I also fix that one above?